apis/python/src/tiledbsoma/io/_registration/signatures.py

```diff
@@ -72,7 +72,10 @@ def _string_dict_from_pandas_dataframe(
         df.reset_index(inplace=True)
         if default_index_name in df:
             if "index" in df:
-                df.drop(columns=["index"], inplace=True)
+                # Avoid warning:
+                # "A value is trying to be set on a copy of a slice from a DataFrame"
+                # df.drop(columns=["index"], inplace=True)
```

Comment:
To make sure I understand this comment: we're *not* doing `inplace=True` to avoid this warning? It might be good to phrase this as something like "We don't do this in-place to avoid the 'A value is trying ..."